CMakeLists.txt

```diff
@@ -464,7 +464,7 @@ endif()
 # http://standards.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html
 # http://standards.freedesktop.org/shared-mime-info-spec/shared-mime-info-spec-latest.html
 if(ENABLE_QT AND UNIX AND NOT APPLE)
-    install(FILES "${PROJECT_SOURCE_DIR}/dist/citra-qt.desktop"
+    install(FILES "${PROJECT_SOURCE_DIR}/dist/azahar.desktop"
```

Comment:
As a note: I've changed here, because the desktop file is unique in both and the executable pointed in both is `azahar`

Reply:
I also reverted the original name change in Doxygen with https://github.com/azahar-emu/azahar/pull/614/commits/f3361b7d430ea3ce37ffd9b3093b0ec4c7e9003a because the .6 files aren't generated in the source repo anyways, this need a fix upstream